rollup.config.js

```diff
@@ -60,8 +88,10 @@ export default cliargs => [
       file: 'dist/video.js',
       name: 'videojs',
       strict: false,
-      banner
+      banner,
+      globals: globals.browser
```

Comment:
do we want to specify globals/external for the UMD files? Since we want the dependencies to be included for those or does resolve and cjs plugins take care of it?

Reply:
the only modules we want to exclude in the umd are `global` since those don't have to be a module and can just be the actual global in the browser